docker/configure.py

```diff
@@ -15,14 +15,15 @@
 postgres_password = ''.join(secrets.choice(chars) for i in range(16))
 
 config = {
-    'DEBUG': False,
+    'DOMAIN': input('Domain name: '),
+    'DEBUG': True if input('Debug (y/N): ') in ['y', 'yes'] else False,
     'DJANGO_SECRET_KEY': django_secret_key,
     'POSTGRES_PASSWORD': postgres_password,
-    'EMAIL_HOST': '',
-    'EMAIL_PORT': '',
-    'EMAIL_USER': '',
-    'EMAIL_PASSWORD': '',
-    'EMAIL_FROM': '',
+    'EMAIL_HOST': input('Email host (smtp.example.net): '),
+    'EMAIL_PORT': input('Email port (465): '),
+    'EMAIL_USER': input('Email user (to login): '),
```

Comment:
I think login (noun) should be log in (verb)?

Reply:
changed